trade-finance-manager-api/src/graphql/reducers/mappings/gef-facilities/mapGefFacilities.js

```diff
@@ -3,10 +3,15 @@ const mapGefFacility = require('./mapGefFacility');
 const mapGefFacilities = (dealSnapshot, dealTfm) => {
   const { facilities } = dealSnapshot;
 
-  const mappedFacilities = facilities.map((facility) =>
-    mapGefFacility(facility, dealSnapshot, dealTfm));
+  // Map facilities if only they exists
+  if (facilities?.length) {
```

Comment:
do we need optional chaining here? or can it just be facilities.length

Reply:
Facilities can be `unidentified` in certain occasions.